### ISSUE TITLE
[Quest][Northrend] Leave our mark

### ISSUE THREAD

**Issue by MrAndersonX** [dalaranwow/dalaran-wow#3899]
[//]: # (INFORMATION ABOUT TAGS)
[//]: # (The black and yellow tag is unique. There can only be one label assigned to them - one black and one yellow)
[//]: # (OLD - ... tags should no longer be used)
[//]: # (Priority - ... tags are assigned by team members)

[//]: # (***************************************************************)
[//]: # (** !!!!!!!!!!!!!DON'T DELETE THIS TEMPLATE ELSE YOUR ISSUE WILL BE CLOSED!!!!!!!!!!!! **)
[//]: # (***************************************************************)

**Description**:
 At the moment, if 2 or more dead mobs are too close to each other, the player gets an error when trying to place the Ebon Blade Banner upon each of the corpses.

**Current behaviour**: Tell us what happens in detail (Where, When, What, Why, Who)
 There's an error saying "Invalid target" that doesn't allow the player to place more than 1 banner if he kills multiple mobs at the same time near each other.

**Expected behaviour**: Tell us what should happen instead
 The player should be allowed to place each banner on each corpse even if they have are next to each other.

**Steps to reproduce the problem**:

1.  Take the quest.
2.  Take a bunch of mobs and aoe kill them.
3. 

**Include proofs for this behaviour**

https://postimg.org/gallery/19xvvs28e/

You can research this on several pages like http://wowhead.com or http://wowwiki.wikia.com or http://www.youtube.com/
Issues without proofs will be closed and tag On Hold will be added until the proof is provided.

**Include the ID for the game objects, npcs (creatures,pets,minions), spells, items, quests, instances, zones, achievements, skills**

You can research this on several pages like http://wotlk.openwow.com or http://wowhead.com/

**Include Screenshots from the issue if necessary**

**Comment by Rushor** [comment]
well the condition that you cannot place the Ebon Blade Banner if there is already a Ebon Blade Banner in range was added to prevent completing the quest on the same mob :/

**Comment by Rushor** [comment]
http://wotlk.openwow.com/quest=12995

**Comment by Rushor** [comment]
DB/Quest: Leave our mark - Enable marking if creatures are close together

* closes https://github.com/dalaranwow/dalaran-wow/issues/3899
* commit: ba47ac0eca9e61cf98f670c663b9bf738bea372f

The issue was fixed by a developer and will now be tested on the Public Test Realm (PTR). The changes/fixes will be applied with the NEXT changelog on the liverealm.
This means that the issue exists in game at the moment i wrote this message.
Only once the issue was marked with "Fixed - On Live Server", you can find the fix on you server.
Please watch http://www.dalaran-wow.com/news/ to see when the changelog and the fix was applied.
